Source/Hurl.Settings/Views/RulesetPage.xaml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="Hurl.Settings.Views.RulesetPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Hurl.Settings.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:model="using:Hurl.Library.Models"
+    xmlns:toolkit="using:CommunityToolkit.WinUI.Controls"
+    xmlns:uc="using:Hurl.Settings.Controls"
+    xmlns:vm="using:Hurl.Settings.ViewModels"
+    mc:Ignorable="d">
+    <Page.DataContext>
+        <vm:RulesetPageViewModel x:Name="ViewModel" />
+    </Page.DataContext>
+
+    <StackPanel MaxWidth="800" Spacing="16">
+        <StackPanel>
+            <TextBlock
+                Margin="2,0,0,4"
+                Style="{StaticResource BodyStrongTextBlockStyle}"
+                Text="Rulesets" />
+
+            <toolkit:SettingsCard Description="Each Ruleset is a collection of Rules" Header="Create new Ruleset">
+                <toolkit:SettingsCard.HeaderIcon>
+                    <FontIcon Glyph="&#xE710;" />
+                </toolkit:SettingsCard.HeaderIcon>
+                <Button
+                    Width="80"
```

Comment:
Advice: Be careful to hardcode sizes like this. Either let the control grow with the text, or set MinWidth to 120.
Also, you might want to think about including this button in the following SettingsExpander. It's all related. Idea to compare: Windows Settings > Themes.

Reply:
> Also, you might want to think about including this button in the following SettingsExpander. It's all related

While that sounds reasonable. I still prefer it this way since it doesn't look crowded.